### PR TITLE
turn ON semantic validation as a required test for PR_ONLY=true

### PR DESCRIPTION
**Pull request by amarzavery**
@veronicagg, @olydis @fearthecowboy - Please review as discussed.

### CODE REVIEW

**Comment by amarzavery** [comment]
We can always revert this change, if we see this is getting problematic. Although it would be nice if we can fix validator bugs to ensure that semantic validator keeps on providing its goodness.

**Comment by marstr** [comment]
Given that this PR seems to make the same change as #1931 and some, I think we should close the other and keep this one.

File: .travis.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ matrix:
     - env: MODE=semantic PR_ONLY=false
     - env: MODE=model PR_ONLY=false
     - env: MODE=linter PR_ONLY=true
-    - env: MODE=semantic PR_ONLY=true
+    # - env: MODE=semantic PR_ONLY=true
```
Comment (marstr) [inline .travis.yml]: I'd prefer to just delete this line instead of commenting it out. We don't want to accidentally have it present in both categories.

**Review by marstr — CHANGES_REQUESTED** [review]
nit

**Comment by amarzavery** [comment]
closing in favor of #1931

**Pull request closed without merging**